test/rest-smoketest.sh

```diff
@@ -219,13 +251,26 @@ dotest "200::*::*" "/ns-${RUNKEY}/foo2/obj1;upload" --head
 dotest "404::*::*" "/ns-${RUNKEY}/foo2;upload"
 dotest "405::*::*" "${upload}/0"
 dotest "405::*::*" "${upload}/0" --head
-split -b 1024 -d "$0" /tmp/parts-${RUNKEY}-
```

Comment:
Why are these blocks of code duplicated when the only difference is the chunk size?  The preceding code that generated different test data should just set a different CHUNK_SIZE or similar variable and then use it in one common codepath.